test/src/pelz_test.c

```diff
@@ -22,10 +22,10 @@
 
 #include "sgx_urts.h"
 #include "pelz_enclave.h"
-#include "pelz_enclave_u.h"
+#include ENCLAVE_HEADER_UNTRUSTED
```

Comment:
The test files should probably explicitly include ```pelz_enclave_*.h``` since that's what they always use.